### PR TITLE
[zh-cn]: create the translation of `declarativeNetRequest.RuleAction` type

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/ruleaction/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: declarativeNetRequest.RuleAction
+slug: Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/RuleAction
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/ruleaction/index.md:8]: 少了一行描述，请补上

https://github.com/mdn/content/blob/5f393e95a212ce0660d4322699dc1456f4cbf76c/files/en-us/mozilla/add-ons/webextensions/api/declarativenetrequest/ruleaction/index.md?plain=1#L10

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
有一处需要注意的地方

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/ruleaction/index.md (hunk 1 above)
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/ruleaction/index.md:8]:
```suggestion
{{AddonSidebar}}

{{WebExtAPIRef("declarativeNetRequest.Rule")}} 的 `action` 属性表明了在匹配到规则时采取的相应动作的细节。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**